src/config/session.c

```diff
@@ -74,14 +77,53 @@ read_environment_file(const char *filename)
 }
 
 static void
-update_activation_env(bool initialize)
+backend_check_drm(struct wlr_backend *backend, void *is_drm)
 {
+	if (is_drm && wlr_backend_is_drm(backend)) {
+		*(bool *)is_drm = true;
+	}
+}
+
+static bool
+should_update_activation(struct server *server)
+{
+	assert(server);
+
+	static const char *act_env = "LABWC_UPDATE_ACTIVATION_ENV";
+	char *env = getenv(act_env);
+	if (env) {
+		/* Respect any valid preference from the environment */
+		int enabled = parse_bool(env, -1);
```

Comment:
Should we add "0" and "1" to `src/common/parse-bool.c` ?

Reply:
Good idea. Seems right for environment variables in particular.